code/SS_Datetimezone.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Class SS_Datetimezone
+ *
+ * Adds customisable timezones to the nice method on {@link SS_Datetime}.
+ */
+class SS_Datetimezone extends SS_Datetime
+{
+    /**
+    * Returns the date in the raw SQL-format specific to a given timezone passed from the Member class, e.g. “2006-01-18 16:32:04”
+    */
+    public function Nice() {
+        if($timestamp = $this){
```

Comment:
You might be better off doing something like this actually:

```
$timestamp = new DateTime($this->value, new DateTimeZone(date_default_timezone_get()));
```

then the whole `if` section is unnecessary. I haven't tested it, but think it should work.